arch-design/multicloud/BucketManagement.md

```diff
@@ -0,0 +1,369 @@
+# Bucket management in soda-multicloud
+
+
+**Author(s)**: [Pravin Ranjan](https://github.com/PravinRanjan10), [Vineela Pachchipulusu]()
+
+
+## Motivation and background
+
+In the cloud world, every cloud vendor provides storage space to users. Generally in terms of bucket and folder. A user can store the data/objects in a specific bucket or folder.
+
+SODA multi-cloud(gelato) is a project which manages multiple cloud vendors from one place. And also interfaces to use cloud services. So it is important for soda to provide bucket management support for various cloud vendors. So that user can create and manage buckets from one place(soda multi-cloud).
```

Comment:
Considering this enhancement, Will Bucket Management for SODA multi-cloud keep the virtual bucket terminology as per the existing design Or they will be any updates in the design?

Will the virtual buckets in SODA multi-cloud (folders in actual cloud backend) be impacted with this enhancement?

Reply:
In new design, Bucket will be the actual bucket(I mean, not folder). And also there will be no impact on current folder(virtual bucket) design.